reference/sectionC_interpretation.rst

```diff
@@ -5,15 +5,21 @@
 ===========================================
 Section C: Interpretation
 ===========================================
+
 .. marker_interpretation_of_imports_start
 
 .. _specC_interpretation_of_imports:
 
 Interpretation of imports
 -------------------------
 
-#. Each :code:`import` element present in a :ref:`CellML infoset<specA_cellml_infoset>` (the importing infoset) SHALL define a new and separate instance of the CellML infoset referenced by the :code:`href` attribute (the imported infoset).
-   See :ref:`Units reference<specC_units_reference>` and    :ref:`Component reference<specC_component_reference>` for the specifics    of importing units and components.
+#. Each :code:`import` element present in a :ref:`CellML infoset<specA_cellml_infoset>` (the importing infoset) SHALL define a new and distinct instance of the CellML infoset which is referenced by the :code:`href` attribute (the imported infoset). This imported infoset instance SHALL provide the context in which the child of an import element (a :code:`units` or :code:`component` element) is defined and interpreted.
+
+#. The importing of an element SHALL be considered recursively such that items referenced or imported by the :code:`import` element child (a :code:`units` or :code:`component` element) are themselves available to that :code:`import` element child.
```

Comment:
"referenced or imported by" isnt' right. You reference things that are in the model either through definition or importing

Reply:
Hmm. If I have a model that imports a units item from another model, I want to be able to say that the imported units will drag anything it needs along with it, whether those things are imported or defined in the imported model.  I wanted "referenced" to mean "locally requested by the imported item" ... is there a better term or way of saying that?

Reply:
Cont. on #86!